tests/rptest/services/redpanda.py

```diff
@@ -1382,8 +1382,17 @@ def restart_nodes(self,
                       nodes,
                       override_cfg_params=None,
                       start_timeout=None,
-                      stop_timeout=None):
+                      stop_timeout=None,
+                      rolling_restart_nodes=False):
```

Comment:
`rolling_restart` parameter was fine. i was thinking that we'd add a new method and have `restart_nodes(self)` and `restart_nodes_rolling(self)`. delegating immediately to some other code based on the parameter makes it seem like it already is a separate method. wdyt?

Reply:
Yea I guess there is enough of a disparity between a hard restart and rolling restart that it warrants its own method. I originally thought it might be useful to keep them together eg for the sake of parameterizing tests or somesuch, but I think the more likely future parameterizations would be on arguments to a rolling restart (e.g. numb to restart at a time, etc), rather than to a generic restart method.